e2e/harmony/installation/policies-order.e2e.ts

```diff
@@ -0,0 +1,46 @@
+import { resolveFrom } from '@teambit/toolbox.modules.module-resolver';
+import { expect } from 'chai';
+import fs from 'fs-extra';
+import path from 'path';
+import Helper from '../../../src/e2e-helper/e2e-helper';
+
+describe('default peer dependency of the env declared by the env has higher priority than the same dependency from other sources', function () {
```

Comment:
I'm not sure this describes really describes correctly the "it" below
I think you can make it just a policies order or something like this

also may be worth adding a test that makes sure that if you have react X 
and you get other react version from the teambit.envs/env yours is stronger.